### ISSUE TITLE
error in loss functions

### ISSUE THREAD

**Issue by platoTao** [yyyouc/DANE#3]
`        nn_x_loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=self.w + tf.eye(tf.shape(self.neg_w)[0]),
                                                            logits=pre_logit_nn_x) \
                    - tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.ones_like(tf.diag_part(pre_logit_nn_x)),
                                                              logits=tf.diag_part(pre_logit_nn_x))
        nn_z_loss = tf.nn.sigmoid_cross_entropy_with_logits(labels=self.w + tf.eye(tf.shape(self.neg_w)[0]),
                                                            logits=pre_logit_nn_z) \
                    - tf.nn.sigmoid_cross_entropy_with_logits(labels=tf.ones_like(tf.diag_part(pre_logit_nn_z)),
                                                              logits=tf.diag_part(pre_logit_nn_z))
        first_order_loss = tf.reduce_mean(pp_x_loss + pp_z_loss + nn_x_loss + nn_z_loss)`

the labels should be self.neg_w + tf.eye(tf.shape(self.neg_w)[0])
